cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -836,3 +838,52 @@ func TestSimulateTransactionBumpAndRestoreFootprint(t *testing.T) {
 	assert.NoError(t, err)
 	sendSuccessfulTransaction(t, client, sourceAccount, tx)
 }
+
+func TestCLI(t *testing.T) {
```

Comment:
This name is a bit too generic

Reply:
Yeah will definitely change once I get it working.